### PR TITLE
HOMS-187 Move from factory_girl to factory_bot

### PR DESCRIPTION
**Pull request by pleaseduck**

### CODE REVIEW

**Comment by sugarfree1** [comment]
https://github.com/thoughtbot/factory_bot/blob/4-9-0-stable/UPGRADE_FROM_FACTORY_GIRL.md

File: Gemfile [hunk 1]
```diff
@@ -57,7 +57,7 @@ group :development do
 end
 
 group :development, :test, :staging do
-  gem 'factory_girl_rails'
+  gem "factory_bot_rails"
```
Comment (sugarfree1) [inline Gemfile]: single quotes

File: config/database.yml.bkp [hunk 2]
```diff
@@ -0,0 +1,8 @@
+development:
```
Comment (sugarfree1) [inline config/database.yml.bkp]: Unnecessary file.

**Comment by pleaseduck** [comment]
upd

**Pull request merged**